src/client/app/components/groups/EditGroupModalComponent.tsx

```diff
@@ -918,6 +919,7 @@ export default function EditGroupModalComponent(props: EditGroupModalComponentPr
 	 */
 	function metersToSelectOptions(): SelectOption[] {
 		// In format for the display component for menu.
+		const locale = useAppSelector(selectSelectedLanguage);
```

Comment:
I'm not sure that the uses of a React Hook (useAppSelector) is okay here. Unlike, CreateGroupModalComponent where the language is selected in the top level React component, here it is done in a nested function. While this can be safe it is tricky so I think we should avoid when possible. In this case the call may depend on if you are an admin which might cause issues. For that reason and to be consistent, I think it would be best to do it as in the create code to set a variable in the top level function and then use it in the nested functions. This applies in at least one other place in this file.